fe/fe-core/src/main/java/com/starrocks/connector/iceberg/Partition.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.starrocks.connector.iceberg;
+
+import com.starrocks.connector.PartitionInfo;
+
+public class Partition implements PartitionInfo {
+    private final long modifiedTime;
```

Comment:
it's almost same as HivePartition, why invent another wheel ?

Reply:
not same, hive partition has more indo which iceberg meta data do not have, and iceberg partition will not only has modifiedTime, it will have more info in the future